layouts/partials/common/opengraph.html

```diff
@@ -47,14 +47,14 @@
 {{ end }}{{ end }}
 {{- end }}
 
-{{- /* Deprecate site.Social.facebook_admin in favor of site.Params.social.facebook_admin */}}
+{{- /* Deprecate site.Social.facebook_admin in favor of */}}
```

Comment:
Comment disappeared

Reply:
It's a comment regarding the issue we try to solve, we should remove the other one as well.

Reply:
Works for me. I just noticed it got cut off :).